frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/basicTicketImage/ticketImageAction/ticketImageAction.component.tsx

```diff
@@ -15,21 +15,21 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 import { ProjectsHooksSelectors } from '@/v5/services/selectorsHooks';
-import { isUndefined } from 'lodash';
 import { Action } from './ticketImageAction.styles';
 
 type TicketImageActionProps = {
 	onImageChange?: any;
 	children: any;
 	disabled?: boolean;
+	onClick?: (...args) => void;
 };
 export const TicketImageAction = ({
 	onImageChange,
 	disabled: disabledInput,
 	...props
 }: TicketImageActionProps) => {
 	const { isAdmin } = ProjectsHooksSelectors.selectCurrentProjectDetails();
-	const disabled = isUndefined(disabledInput) ? disabledInput : !isAdmin;
+	const disabled = disabledInput || !isAdmin;
```

Comment:
The reason why this was using `isUndefined(disabledInput)` is so that this component knows whether or not `disabledInput` has been set from the parent. If you only use `disabledInput || !isAdmin`, the parent component will not be able to set the action as "enabled" because the `isAdmin` would override that value.

**Example:**
The user (Alice) is not an admin, but the action is `go to viewpoint`, which should be enabled for any users.

If the parent component sets `disabledInput` to `false`, Alice will still not be able to click the action  
> `disabledInput || !isAdmin` = `false || true` = `true`. 

Using the `isUndefined(disabledInput)`, instead, you would still allow Alice to click on that action
> `!isUndefined(disabledInput) ? disabledInput : !isAdmin` = `true ? false : true` = `false`
(**Note**: I forgot to add the `not` in front of `isUndefined`)

Reply:
why are we checking `isAdmin`?   @sanmont3drepo

The current scope of the work we shouldn't be dealing with permissions, let the request go through and let the api endpoint errors out.

Only allow admins to update a ticket is not a desired behaviour... also the permissions mentioned in the scope is not correct, it should be commenter plus 
![image](https://user-images.githubusercontent.com/11945337/204272897-6bcdd3e1-e9b2-4a2f-9966-dc72b1e9af83.png)


I didn't sanity check this when you opened the issue because we agreed we were not dealing with the permissions 😆

Reply:
You are right @carmenfan  we shouldnt worry about this at this point.